src/main/java/mekanism/common/item/ItemModule.java

```diff
@@ -46,9 +49,17 @@ public Rarity getRarity(@Nonnull ItemStack stack) {
     @Override
     public void appendHoverText(@Nonnull ItemStack stack, Level world, @Nonnull List<Component> tooltip, @Nonnull TooltipFlag flag) {
         if (MekKeyHandler.isKeyPressed(MekanismKeyHandler.detailsKey)) {
+            tooltip.add(new TextComponent("Supported by:"));
```

Comment:
Converting this and conflicting to translation keys should be fine, especially if you add/rename `MekanismLang.UPGRADE_DISPLAY` to `MekanismLang.GENERIC_LIST` and then move it to the generic section and use that for the elements being added to each sub category.